src/gentropy/finngen_ukb_meta.py

```diff
@@ -0,0 +1,49 @@
+"""Step to run FinnGen UKB meta-analysis data ingestion."""
+
+from __future__ import annotations
+
+from gentropy.common.per_chromosome import (
+    prepare_va,
+    process_summary_stats_per_chromosome,
+)
+from gentropy.common.session import Session
+from gentropy.datasource.finngen_ukb_meta.study_index import FinngenUkbMetaStudyIndex
+from gentropy.datasource.finngen_ukb_meta.summary_stats import (
+    FinngenUkbMetaSummaryStats,
+)
+
+
+class FinngenUkbMetaStep:
```

Comment:
NIT: the name of the step does not inform what it does.

Reply:
I named it in line with the existing `UkbPppEurStep` for consistency, but I'm open to change it. Do you have a proposal for a possible replacement name?

Reply:
it is ingestion + harmonisation, should be call it strictly `FinngenUkbMetaIngestionStep` ?

Reply:
Now renamed the class as you suggested, thank you!